src/components/activity/screens/ActivityAdd/containers/ActivityAddContainer.js

```diff
@@ -6,23 +6,28 @@ moment.locale('fr');
 
 import ActivityAddView from '../views/ActivityAddView';
 import { addActivity } from '../actions/index';
+import { ActivityTypeEnum } from 'yasav/src/const';
 
 
 class ActivityAddContainer extends React.Component {
 
   constructor(props){
     super(props);
     this.state = {
-      title: "",
-      description: "",
-      date: moment(),
-      type: "content",
-      contentSource: "",
-      eventWhat: "",
-      meetingWho: "",
-      tags: [],
+      isFormValid: false,
       tagInput: "",
-      key: this.props.lastID + 1
+      activity: {
+        title: "",
+        description: "",
+        date: moment(),
+        type: ActivityTypeEnum.CONTENT,
+        contentSource: "",
+        eventWhat: "",
+        meetingWho: "",
+        tags: [],
+        key: this.props.lastID + 1,
+      },
```

Comment:
I think we should also rename "key" into "id" if possible to ensure coherence with the DB.

Reply:
It'll be fixed in the next pr by @sioufia